package.json

```diff
@@ -40,7 +40,7 @@
     "vscode-languageserver-types": "^3.16.0",
     "vscode-nls": "^5.0.0",
     "vscode-uri": "^3.0.2",
-    "yaml": "2.0.0-8"
+    "yaml": "2.0.0-9"
```

Comment:
Since this line is touched anyway, I think you might as well update update to [2.0.0-10](https://github.com/eemeli/yaml/releases/tag/v2.0.0-10). It should be a trivial change.

Reply:
OK, I plan to do it anyway